src/test/kotlin/com/ampnet/userservice/controller/RegistrationControllerTest.kt

```diff
@@ -106,6 +111,48 @@ class RegistrationControllerTest : ControllerTestBase() {
         }
     }
 
+    @Test
+    fun mustGetErrorIfReCaptchaReturnsError() {
+        suppose("ReCAPTCHA verification failed") {
+            Mockito.`when`(reCaptchaService.validateResponseToken(testUser.reCaptchaToken))
+                .thenAnswer { throw ReCaptchaException("ReCAPTCHA verification failed") }
+        }
+
+        verify("Controller will return ReCaptcha error code") {
+            val requestJson = generateSignupJson()
+            val result = mockMvc.perform(
+                post(pathSignup)
+                    .content(requestJson)
+                    .contentType(MediaType.APPLICATION_JSON)
+            )
+                .andExpect(status().isBadRequest)
+                .andExpect(content().contentType(MediaType.APPLICATION_JSON))
+                .andReturn()
+            verifyResponseErrorCode(result, ErrorCode.REG_RECAPTCHA)
+        }
+    }
+
+    @Test
+    fun mustGetErrorIfReCaptchaReturnsLowScore() {
+        suppose("ReCAPTCHA verification failed") {
+            Mockito.`when`(reCaptchaService.validateResponseToken(testUser.reCaptchaToken))
+                .thenAnswer { throw ReCaptchaException("ReCAPTCHA score is too low") }
+        }
+
+        verify("Controller will return ReCaptcha error code") {
+            val requestJson = generateSignupJson()
+            val result = mockMvc.perform(
+                post(pathSignup)
+                    .content(requestJson)
+                    .contentType(MediaType.APPLICATION_JSON)
+            )
+                .andExpect(status().isBadRequest)
+                .andExpect(content().contentType(MediaType.APPLICATION_JSON))
+                .andReturn()
+            verifyResponseErrorCode(result, ErrorCode.REG_RECAPTCHA)
+        }
+    }
```

Comment:
This test is not needed. Covers the same case as the one above: mustGetErrorIfReCaptchaReturnsError.